scripts/generate-changelog.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const replace = require('replace-in-file')
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
+const spawn = util.promisify(require('child_process').spawn)
+
+const name = require('emoji-name-map')
+
+const lernaConfig = require('../lerna.json')
+
+const CHANGELOG_FILE = 'CHANGELOG.md'
+
+async function main() {
+  const lastTagHash = await executeCommand('git rev-list --tags --max-count=1')
+  const lastTagName = await executeCommand(`git describe --tags ${lastTagHash}`)
+
+  const oldTag = lastTagName.replace(/\n$/, '')
+  const commits = await executeCommand(`git log ${oldTag}..HEAD --pretty=format:"- %s"`)
```

Comment:
I am not sure to understand why the `\n` is an issue for the log command but not for the describe 🤔 

If it is needed, could we inline `oldTag` variable or name it to be consistent with `lastTagXXX`?

Reply:
The `\n` is always an issue if you pass it to the next command.
The describe command returns the tagName with an extra newline.

Reply:
Also, just use `.trimEnd()` ^^